doc/assessment.md

```diff
@@ -84,6 +84,10 @@ An assessment is broken down in to a list of zones, like this:
 
 * An assessment question can be specified by either a single `id` or by a list of alternatives, in which case one or more of these alternatives is chosen at random. Once the question `id` is determined, then a random variant of that question is selected. Question alternatives inherit the points of their parent group, if specified.
 
+* If a zone has `maxPoints`, then, of the points that are awarded for answering questions in this zone, at most `maxPoints` will count toward the total points.
+
+* If a zone has `numberGrade`, then, of the questions in this zone, only `numberGrade` with the highest number of awarded points will count toward the total points.
```

Comment:
I agree with @nwalters512 in that I'm not wild about the name of `numberGrade`. Perhaps it's because we are grading all of them, but only using some of them? I'm aware we are probably well into bike-shedding territory here.

Looking at the HTML we show to users, we say "maximum X points", which sounds a lot like `maxPoints`. Then we say "best N questions", so maybe we the attribute should be `bestQuestions`?

Reply:
done - went with `bestQuestions`